samples/Webserver/WebServer.DI/webserver-cert.crt

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
you've checked in this cert but you're not using it. I guess it's because you've been testing it with https but in your sample, you're using http. So either use it, either remove it

Reply:
Actually, I copied all files from other sample and forgot to delete it. Removed.